react/isItBurnt/client/src/views/searchRes.js

```diff
@@ -10,25 +10,25 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 
 const SearchRes = ({recipe, recipeInfo, value}) => {
 
-    const [recipeName, setRecipeName] = useState(value)
-    useEffect(()=> setRecipeName(recipeInfo.title), [recipeInfo.title])
-    const [recipeImage, setRecipeImage] = useState(value)
-    useEffect(()=> setRecipeImage(recipeInfo.image), [recipeInfo.image])
-    const [recipeUrl, setRecipeUrl] = useState(value)
-    useEffect(()=> setRecipeUrl(recipeInfo.sourceUrl), [recipeInfo.sourceUrl])
-    const [recipeUrlName, setRecipeUrlName] = useState(value)
-    useEffect(()=> setRecipeUrlName(recipeInfo.sourceName), [recipeInfo.sourceName])
-    const [recipeSummary, setRecipeSummary] = useState(value)
-    useEffect(()=> setRecipeSummary(recipeInfo.summary), [recipeInfo.summary])
-    const [recipeInstructions, setRecipeInstructions] = useState(value)
-    useEffect(()=> setRecipeInstructions(recipeInfo.instructions), [recipeInfo.instructions])
+    // const [recipeName, setRecipeName] = useState(value)
+    // useEffect(()=> setRecipeName(recipeInfo.title), [recipeInfo.title])
+    // const [recipeImage, setRecipeImage] = useState(value)
+    // useEffect(()=> setRecipeImage(recipeInfo.image), [recipeInfo.image])
+    // const [recipeUrl, setRecipeUrl] = useState(value)
+    // useEffect(()=> setRecipeUrl(recipeInfo.sourceUrl), [recipeInfo.sourceUrl])
+    // const [recipeUrlName, setRecipeUrlName] = useState(value)
+    // useEffect(()=> setRecipeUrlName(recipeInfo.sourceName), [recipeInfo.sourceName])
+    // const [recipeSummary, setRecipeSummary] = useState(value)
+    // useEffect(()=> setRecipeSummary(recipeInfo.summary), [recipeInfo.summary])
+    // const [recipeInstructions, setRecipeInstructions] = useState(value)
+    // useEffect(()=> setRecipeInstructions(recipeInfo.instructions), [recipeInfo.instructions])
 
-    // const [recipeName, setRecipeName] = useState(recipeInfo?.title)
-    // const [recipeImage, setRecipeImage] = useState(recipeInfo?.image)
-    // const [recipeUrl, setRecipeUrl] = useState(recipeInfo?.sourceUrl)
-    // const [recipeUrlName, setRecipeUrlName] = useState(recipeInfo.sourceName)
-    // const [recipeSummary, setRecipeSummary] = useState(recipeInfo.summary)
-    // const [recipeInstructions, setRecipeInstructions] = useState(recipeInfo.instructions)
+    const [recipeName, setRecipeName] = useState(recipeInfo?.title)
+    const [recipeImage, setRecipeImage] = useState(recipeInfo?.image)
+    const [recipeUrl, setRecipeUrl] = useState(recipeInfo?.sourceUrl)
+    const [recipeUrlName, setRecipeUrlName] = useState(recipeInfo?.sourceName)
+    const [recipeSummary, setRecipeSummary] = useState(recipeInfo?.summary)
+    const [recipeInstructions, setRecipeInstructions] = useState(recipeInfo?.instructions)
```

Comment:
the `?` is to check if the property value in recipeInfo is undefined. If it is undefined then the recipeName, recipeImage, recipeUrl...etc will be set to undefined and not error out.